apps/crn-server/test/integration/reminders.integration-test.ts

```diff
@@ -200,8 +200,8 @@ describe('Reminders', () => {
 
       app = AppHelper(() => loggedInUser);
     });
-
-    describe('Published Reminder', () => {
+    // eslint-disable-next-line jest/no-disabled-tests
```

Comment:
maybe lets merge this
https://github.com/yldio/asap-hub/pull/3899
then rebase?

Reply:
yes sure

Reply:
this was so i could get it deployed, test it and see the logs